README.md

```diff
@@ -1,54 +1,106 @@
-xblock-group-project-v2
-=======================
+# xblock-group-project-v2
+
 
 This XBLock is experimental reimplementation of [Group Project XBlock](https://github.com/edx-solutions/xblock-group-project).
 
 It does *not* work properly in LMS yet.
 
-Configuration variables
------------------------
+## Features
 
-This block uses following configuration variables in the ``XBLOCK_SETTINGS``:
+* Structure - group project is organized around activities and stages. 
+    * Activities are the larger chunks, used to distinguish between logically bound parts of group project. Activities 
```

Comment:
@e-kolpakov ... "used to distinguish between different parts of a group project".